README.md

```diff
@@ -2,3 +2,98 @@
 
 [![Build Status](https://github.com/JuliaEarth/GeoTIFF.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/JuliaEarth/GeoTIFF.jl/actions/workflows/CI.yml?query=branch%3Amain)
 [![Coverage](https://codecov.io/gh/JuliaEarth/GeoTIFF.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/JuliaEarth/GeoTIFF.jl)
+
+Load and save GeoTIFF files simply and efficiently in pure Julia.
+
+## Installation
+
+Get the latest stable release with Julia's package manager:
+
+```
+] add GeoTIFF
+```
+
+## Usage
+
+### Loading GeoTIFF files
+
+The `GeoTIFF.load` function loads the TIFF image and the GeoTIFF metadata:
+
+```julia
+julia> geotiff = GeoTIFF.load("utm.tif")
+100×100 GeoTIFF.GeoTIFFImage{...}:
+ RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  …  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)
+ RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)     RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)
+ RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)     RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)
+ ⋮                                                                               ⋱                                          
+ RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)     RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)
+ RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)     RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)  RGB(1.0, 1.0, 1.0)
+```
+
+Use the `GeoTIFF.metadata` to get the metadata object:
+
+```julia
+julia> metadata = GeoTIFF.metadata(geotiff)
+GeoTIFF.Metadata(...)
+```
+
+GeoTIFF.jl defines several utilities to easily retrieve metadata information:
+
+```julia
+julia> GeoTIFF.rastertype(metadata) == GeoTIFF.PixelIsArea
+true
+
+julia> GeoTIFF.modeltype(metadata) == GeoTIFF.Projected2D
+true
+
+julia> GeoTIFF.epsgcode(metadata) |> Int # GeoTIFF uses UInt16 for integer values
+32617
+```
+
+### Saving GeoTIFF files
+
+The `GeoTIFF.save` function can be used to save tiff images, color arrays,
+or channel arrays into new GeoTIFF files with given metadata:
+
+```julia
+julia> using TiffImages, Colors
+
+julia> tiff = TiffImages.load("test.tif");
+
+julia> GeoTIFF.save("tiff_img.tiff", tiff) # default metadata
```

Comment:
This example doesn't need to involve the TiffImages.jl internals. Please assume that the geotiff file is already loaded into memory, and explain how to save it with GeoTIFF.save. You can also explain how to convert from a simple TIFF to a GeoTIFF with the addition of metadata.